rdrf/rdrf/templates/rdrf_cdes/radio_select.html

```diff
@@ -1,17 +1,22 @@
 {# Adapted from 'django/forms/widgets/multiple_input.html' #}
 {% with id=widget.attrs.id %}
-  <ul{% if id %} id="{{ id }}"{% endif %} class="container-fluid radio-select {% if widget.attrs.class %}{{ widget.attrs.class }}{% endif %}">
```

Comment:
I'm not sure if it's intentional (and it's only a small thing), but the highlighting of the selected options seems to now be applied to the whole grouping of radios. See the difference between the radios and checkboxes here:

![image](https://github.com/eresearchqut/trrf/assets/60163872/5d0a0640-3f8b-4d1c-89a4-1a30b373ac91)